manifests/config.pp

```diff
@@ -6,11 +6,20 @@
 class icingadb::config {
   assert_private()
 
+  $stdlib_version  = $icingadb::globals::stdlib_version
   $conf_dir        = $icingadb::globals::conf_dir
   $user            = $icingadb::globals::user
   $group           = $icingadb::globals::group
   $redis_tls_files = $icingadb::redis_tls_files
   $db_tls_files    = $icingadb::db_tls_files
+  $config_file     = inline_epp('<%= $preamble %><%= $content -%>', {
```

Comment:
Don't do this please....

Reply:
Only the inline is the problem or a template file is also a no go?